public/js/module/admin/regionCheck.js

```diff
@@ -187,17 +187,14 @@ define([
         updateRegionAbort: function () {
             if (this.ownRegionsDeffered) {
                 this.ownRegionsDeffered.reject();
-                this.ownRegionsDeffered = null;
```

Comment:
Reason for change here - rejects will be picked by .always where variable is nulled.